overlord/state/task.go

```diff
@@ -27,6 +27,12 @@ import (
 	"github.com/snapcore/snapd/logger"
 )
 
+// Keys used to insert custom data by some tasks
+const (
+	KernelCmdlineAppendTaskKey          = "cmdline-append"
+	KernelDangerousCmdlineAppendTaskKey = "dangerous-cmdline-append"
+)
```

Comment:
as discussed we do not do this, if we did we would have to agree to do it across everything and this would still be the wrong place. for now I would just do as we did so far and repeat the strings in the relevant packages

Reply:
Changed now